server/auth/jwt.go

```diff
@@ -60,11 +60,16 @@ func (t *tokenJWT) info(ctx context.Context, token string, rev uint64) (*AuthInf
 	})
 
 	if err != nil {
-		t.lg.Warn(
-			"failed to parse a JWT token",
-			zap.String("token", token),
-			zap.Error(err),
-		)
+		jwtErr := err.(*jwt.ValidationError)
```

Comment:
Is err always expected to be *jwt.ValidationError? Would it not panic if it's not?

Can you handle case when it's not and add a test to confirm it?

Reply:
yeah we shouldn’t have the assumption, let me handle other cases and add tests. Thanks for pointing out that.

Reply:
On the second thought, I think that having an assumption that `jwt.ValidationError` is the only error such that etcd shouldn't print a token might not be good. Such an assumption can be broken easily when the library is upgraded. So I simply updated logging code for hiding a token. How do you think @serathius ?